src/apps/frontend/app.global.css

```diff
@@ -0,0 +1,29 @@
+/* add your global imports and css rules her */
```

Comment:
Where is this coming from?

I believe with baseweb, there should not be any need to write css like this, except when we work with 3rd part lib and we need to override some of the css directly (which again should be discouraged).